llm_toolkit/output_parser.py

```diff
@@ -44,24 +44,33 @@ def parse_args() -> argparse.Namespace:
   return args
 
 
+def _parse_code_block_by_marker(lines: list[str], start_marker: str,
+                                end_marker: str) -> list[str]:
+  """Parses code block lines based on markers."""
+  block, in_block = [], False
```

Comment:
nit: just have two separate lines for these. it's clearer.

Reply:
Done, thanks for the quick review!